c2pa_c/src/signer_info.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License,
+// Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+// or the MIT license (http://opensource.org/licenses/MIT),
+// at your option.
+// Unless required by applicable law or agreed to in writing,
+// this software is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR REPRESENTATIONS OF ANY KIND, either express or
+// implied. See the LICENSE-MIT and LICENSE-APACHE files for the
+// specific language governing permissions and limitations under
+// each license.
+
+use c2pa::{create_signer, Signer, SigningAlg};
+use serde::Deserialize;
+
+use crate::{Error, Result};
+
+/// SignerInfo provides the information needed to create a signer
+/// and sign a manifest.
+///
+/// The signer is created from the signcert and pkey fields.
```

Comment:
Why document these in prose here rather than adjacent to the fields?

Also I notice a disconnect between `tsa_url` in the doc comment at `ta_url` in the actual struct.